sale_order_lot_selection/models/sale.py

```diff
@@ -42,7 +42,8 @@ def get_move_from_line(self, line):
         lot_count = 0
         for p in line.order_id.picking_ids:
             for m in p.move_lines:
-                move_line_id = m.move_line_ids.filtered(lambda line: line.lot_id)
+                move_line_id = \
+                    m.move_line_ids.filtered(lambda line: line.lot_id)
```

Comment:
These formatting changes are not needed.
FYI we now tolerate up to 88 character per line, the default for black.

Reply:
Hi,
I would like to share with us the incidence that occurs to me when creating an order with the 'lot_id' was empty ... and when odoo creating the 'delivery' it does not assign 'lot_id' but the quantity is correct, I think that by default odoo should choose the oldest. Is it right?
On the other hand, if the 'lot_id' is defined from the order, the behavior is correct. 
Could you help me?

Thank you in advance,